src/mapex/write_parsed_mappings.py

```diff
@@ -184,8 +185,8 @@ def write_parsed_mappings_stix(parsed_mappings, filepath):
         "type": "bundle",
         "id": f"bundle--{bundle_uuid}",
         "spec_version": "2.1",
-        "created": datetime.now().isoformat(),
-        "modified": datetime.now().isoformat(),
+        "created": datetime.now(timezone.utc).isoformat().replace("+00:00", "Z"),
```

Comment:
This would be a good time to refactor the way we set the time of these objects. To keep the file consistent, lets fetch the time once, at the start of the function, and then use that time across all newly created STIX objects. This way, all STIX objects will appear to be created at once.